<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimControllerAdvanceTime</class>
 <widget class="QFrame" name="SimControllerAdvanceTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Advance to time:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="UtQtTimeValueEdit" name="mTime">
     <property name="placeholderText">
      <string>s, m, h, days, hh:mm:ss[.n],...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtTimeValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
